lib/btrfs_test.pm

```diff
@@ -88,6 +88,14 @@ By updating the lastrun files timestamps, we make sure those routines won't be e
 while tests are running. 
 =cut
 sub cron_mock_lastrun {
+    my $tries = 5;
+    while (script_run(q/ps aux | grep '[s]napper'/) == 0 && $tries > 0) {
+        sleep 30;
+        bmwqemu::diag('Snapper is running in the background...');
+        $tries--;
+    }
+    $tries or bmwqemu::diag('Snapper might be still running in the background');
```

Comment:
This is a bit weird. Let's say, the first iteration the snapper is not running, so `tries=5` and you are still showing the message. You only hide this message in the case we have the 5 iterations in the loop where `tries=0`...

Reply:
It is vice versa. When `$tries` evaluates as falsy the message appears. 
Some something like
```
bmwqemu::diag('Snapper might be still running in the background') unless $tries;
```

Reply:
you are right, I shouldn't do reviews on Saturday :D